packages/dcos-metrics/buildinfo.json

```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/dcos/dcos-metrics.git",
-    "ref": "b68c5e128120d0bf96a13aa11000999170a33ed6",
-    "ref_origin": "1.1.0"
+    "ref": "62702c3396deb5b47f4492a5a39d9dfb6f95ab0b",
+    "ref_origin": "master"
```

Comment:
@collinvandyck @klueska Are we sure we want to use `master` of `dcos-metrics` in 1.9.1? Have there been other significant changes between these two SHAs? I see 64 commits, and they can't all be the backport for DCOS_OSS-925
https://github.com/dcos/dcos-metrics/compare/b68c5e128120d0bf96a13aa11000999170a33ed6...62702c3396deb5b47f4492a5a39d9dfb6f95ab0b
https://github.com/dcos/dcos/pull/1527 is targeted at master/1.10, so we'll probably need a different PR for 1.9.1 that actually shows the component changelog in the PR description.